src/helpers/notification_helpers.ts

```diff
@@ -1,45 +1,143 @@
 import axios from "axios";
 import { cache_manager, logger, translation_manager } from "../managers";
 import { add_audit_record } from "./global_helpers";
-import { messaging } from "./firebase_helpers";
+import { add_document, messaging } from "./firebase_helpers";
 import { MulticastMessage } from "firebase-admin/messaging";
 import { EventFromDevice } from "akeyless-types-commons";
+import { isIccid, isInternationalPhoneNumber, isLongPhoneNumber } from "./phone_number_helepers";
+import { Twilio } from "twilio";
+import { Timestamp } from "firebase-admin/firestore";
+import { v4 as uniqId } from "uuid";
 
-export const send_sms = async (phone_number: string, text: string, entity_for_audit: string): Promise<void> => {
+type SmsFunction = (number: string, text: string, entity_for_audit?: string) => Promise<void>;
```

Comment:
"number" is confusing name for this parameter, especially for ICCID case. May be better "recepient" ?

Reply:
done